docs/run_flask.md

```diff
@@ -0,0 +1,39 @@
+# How to run this flask app
+
+## Create python environment
+
+
+```sh
+# research_lab/
+
+python -m venv env
+source env/bin/activate
+```
+
+You can verify your python by running
+```sh
+which python
+```
+Your terminal should should show as the following 
+```sh
+/Users/.../research_lab/env/bin/python
+```
+
+Make sure your pip is upgraded to the latest one
+```sh
+pip install --upgrade pip
```

Comment:
change this to conda version

Reply:
This .md file should be deleted, I migrated it to setup.md